.github/workflows/test.yml

```diff
@@ -41,14 +49,22 @@ jobs:
   Test-gun1:
     runs-on: ubuntu-latest
     name: OTP ${{matrix.otp}} / Elixir ${{matrix.elixir}} - Gun1
+    continue-on-error: ${{ matrix.experimental }}
```

Comment:
I'm not sure why there is a separate section for gun? It seems like these tests are also run by the previous test run.

Reply:
I remember removing this, and @teamon put it back again.

Check the commit history ... I remember I made a mistake so I am extra careful now

Reply:
It is there to ensure tests pass with gun 1.x too (the default lockfile will use gun 2.x)

There used to be a similar thing for poison 3. 

We could remove it if we drop support for gun 1.x but I’m not sure how big of an issue that would be for tesla users. I think I’d rather keep that CI job than force (or prevent) people to update their deps major versions.

Reply:
Thanks for clarifying

Reply:
Is it reasonable to run the gun 1 tests only on an older Elixir / OTP combination instead of the whole matrix? (e.g. OTP 25.3, Elixir 1.15)

Reply:
Yes